Frontend/ui-customer/src/Services/API.purs

```diff
@@ -3815,12 +3815,18 @@ newtype BookingUpdateRequestDetails = BookingUpdateRequestDetails {
   estimatedFare :: Maybe Number,
   id :: String,
   oldEstimatedDistance :: Maybe Int,
+  errorObj :: Maybe ErrorObj,
```

Comment:
why error obj is added inside success response ? this looks like temporary fix, can you please explain this use case

Reply:
I think it's not actual error but when a driver rejects the request to update the destination location